docs/sphinx/source/whatsnew/v0.8.0.rst

```diff
@@ -3,15 +3,25 @@
 v0.8.0 (Month day, year)
 -------------------------
 
+API Changes with Deprecations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* Moved functions related to inverters from `pvsystem.py` to `inverter.py`.
+  Functions are renamed to follow a more consistent pattern, as follows (:pull:`886`):
+   - :py:func:`pvlib.pvsystem.snlinverter` is now :py:func:`pvlib.inverter.sandia`
+   - :py:func:`pvlib.pvsystem.pvwatts_ac` is now :py:func:`pvlib.inverter.pvwatts`
+   - :py:func:`pvlib.pvsystem.adrinverter` is now :py:func:`pvlib.inverter.adr`
+* Attribute :py:attribute:`modelchain.ModelChain.ac_model` now accepts ``sandia``,
+  ``pvwatts`` and ``adr`` for the inverter models.  (:pull:`886`)
```

Comment:
I'm guessing the vast majority of people set this through the `ac_model` kwarg rather than the attribute. So I would link to ``:py:class:`pvlib.modelchain.ModelChain` `` instead. Also suggest adding `'` within the code blocks.